index.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- Header Section  -->
+
+    <header>
+      <div class="header-grid">
+        <div class="twitter-logo">
+          <img src="/Pictures/icons/Twitter logo.png" alt="" />
+        </div>
+
+        <div class="home-section home-grid">
+          <div class="home-header">Home</div>
+          <div class="stars-icon">
+            <img src="/Pictures/icons/stars.png" alt="Stars icon" />
+          </div>
+        </div>
+
+        <div class="search search-grid">
+          <div class="search-image">
+            <img src="./Pictures/icons/search.png" alt="Search Image" />
+          </div>
+          <div class="Search-twitter">Search Twitter</div>
+        </div>
+        <div class="space1"></div>
+      </div>
+    </header>
+
+    <!-- Main Section  -->
+
+    <main>
+      <div class="main-section-grid">
+        <div class="left-menu">
```

Comment:
Some divs could have a more appropriate tag. I suggest an `<aside>` tag here for example.